xml/admin_administration.xml

```diff
@@ -49,19 +49,39 @@
     </para>
    </listitem>
   </itemizedlist>
-
+  
   <para>
-   Transactional updates are managed by the
-   <command>transactional-update</command> script, which is called once a day.
-   The script checks if any updates are available. If there are any updates to
-   be applied, a new snapshot of the root file system is created and the system
-   is updated using the <command>zypper dup</command>. All updates released to
-   this point are applied. The snapshot is then marked as active and will be
-   used after the next reboot of the system. Ensure that the cluster is
-   rebooted as soon as possible after the update installation is complete,
-   otherwise all changes will be lost.
+   Transactional updates are managed by the <command>transactional-update</command> 
+   script, which is called once a day. The script checks if any updates are 
+   available. If there are any updates to be applied, a new snapshot of the root 
+   file system is created in the background and is updated using 
+   <command>zypper dup</command>. All updates released to this point are applied.
```

Comment:
I know that it was written in the documentation before, but we should not mention "zypper dup", that's an implementation detail which can always change and should not be from interest for an admin.

Reply:
Agreed, removing the mention of zypper.